hbase_sql/src/main/java/com/splicemachine/hbase/BackupUtils.java

```diff
@@ -234,6 +231,12 @@ public static boolean shouldCaptureIncrementalChanges(FileSystem fs,Path rootDir
                             }
                         }
                     }
+                    else if (BackupUtils.existsDatabaseBackup(fs, rootDir)) {
```

Comment:
What happens if there's a table/schema backup going on but there was a previous full backup ? Without this change we would register the HFile and with it we wouldn't. Should we change the check to `if (!shouldRegister && BackupUtils.existsDatabaseBackup(fs, rootDir)` ? Would that create a race condition in the SYSBACKUPS file too?

Reply:
Good point!. Table/schema backup should not write to SYSBACKUPS file. I made changes to ee branch.